home/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("list",views.home_list,name="home_list"),
     path("create",views.home_create,name="home_create"),
-    path("delete",views.home_delete, name="home_delete")
+    path("delete",views.home_delete, name="home_delete"),
+    path("update/<slug:slug>",views.home_update,name="home_update")
```

Comment:
Following SD urls structure, you may put the slug first in url, using this pattern:
      "<slug:slug>/update"
Instead of:
      "update/<slug:slug>"